### ISSUE TITLE
should i do nb_classes=1 or 2 for binary classification finetuning task?

### ISSUE THREAD

**Issue by forkbabu** [EPFL-VILAB/MultiMAE#20]
Hi @roman-bachmann , @dmizr , @amir32002 great paper

Let me know what is better.

**Comment by dmizr** [comment]
Hi @forkbabu ,

If you want to use our classification fine-tuning script without making many changes, you can use `nb_classes=2`.
If you use `nb_classes=1`, you'll need to modify or disable some augmentations like Mixup, and switch the loss to [`BCEWithLogitsLoss`](https://pytorch.org/docs/stable/generated/torch.nn.BCEWithLogitsLoss.html#torch.nn.BCEWithLogitsLoss).
Keep in mind that the fine-tuning script we provide is based on [timm](https://github.com/rwightman/pytorch-image-models) and is designed for tasks like ImageNet classification, so you should check if it's a good fit for your use case

Best, David
